spec.html

```diff
@@ -5546,6 +5717,102 @@ <h1>CopyDataProperties ( _target_, _source_, _excludedItems_ )</h1>
         <p>The target passed in here is always a newly created object which is not directly accessible in case of an error being thrown.</p>
       </emu-note>
     </emu-clause>
+
+    <emu-clause id="sec-privateelementfind" aoid="PrivateElementFind">
+      <h1>PrivateElementFind ( _P_, _O_ )</h1>
+      <p>The abstract operation PrivateElementFind takes arguments _P_ (a Private Name) and _O_ (an Object). It performs the following steps when called:</p>
+      <emu-alg>
+        1. If _O_.[[PrivateElements]] contains a PrivateElement Record whose [[Key]] is _P_, then
+          1. Let _entry_ be that PrivateElement Record.
+          1. Return _entry_.
+        1. Return ~empty~.
+      </emu-alg>
+    </emu-clause>
+
+    <emu-clause id="sec-privatefieldadd" aoid="PrivateFieldAdd">
+      <h1>PrivateFieldAdd ( _P_, _O_, _value_ )</h1>
+      <p>The abstract operation PrivateFieldAdd takes arguments _P_ (a Private Name), _O_ (an Object), and _value_ (an ECMAScript language value). It performs the following steps when called:</p>
+      <emu-alg>
+        1. Let _entry_ be ! PrivateElementFind(_P_, _O_).
+        1. If _entry_ is not ~empty~, throw a *TypeError* exception.
+        1. Append PrivateElement { [[Key]]: _P_, [[Kind]]: ~field~, [[Value]]: _value_ } to _O_.[[PrivateElements]].
+      </emu-alg>
+    </emu-clause>
+
+    <emu-clause id="sec-privatemethodadd" aoid="PrivateMethodAdd">
+      <h1>PrivateMethodAdd ( _method_, _O_ )</h1>
```

Comment:
Nit: PrivateMethodOrAccessorAdd?

Reply:
Done in https://github.com/tc39/ecma262/pull/1668/commits/59e7640757945a845318a93b0695bff7f9ee3162.